signal/cmd/root.go

```diff
@@ -52,11 +63,29 @@ func SetupCloseHandler() {
 }
 
 // InitLog parses and sets log-level input
-func InitLog(logLevel string) {
+func InitLog(logLevel string, logPath string) {
 	level, err := log.ParseLevel(logLevel)
 	if err != nil {
 		log.Errorf("Failed parsing log-level %s: %s", logLevel, err)
 		os.Exit(ExitSetupFailed)
 	}
+
+	if logPath != "" {
```

Comment:
Tested by adding stdout path as -`-log-file /proc/self/fd/1`, but I got this warning when triggering the log rotation:

```time="2021-09-06T21:48:47Z" level=info msg="can't rename log file: rename /proc/self/fd/1 /proc/self/fd/1-2021-09-06T21-48-47.313: no such file or directory"```

Also tested passing an empty string with ("") and that works, but is not intuitive and for Docker, brings complexity to scape characters to avoid making "" nonempty but the file name.

For log file, makes sense to add a keyword that can be used to indicate console log outputs, allowing small changes to our Dockerfile.

Reply:
if you go with the key, we also need to update the docker-compose in the infrastructure_files